src/test/java/hudson/plugins/openid/OpenIdUserPropertyTest.java

```diff
@@ -62,7 +67,9 @@ public void testDisabledRoundtrip() throws Exception {
 
         // submit empty config
         WebClient wc = createWebClient();
-        HtmlPage pg = wc.goTo("user/alice/configure");
+        HtmlPage pg = Jenkins.getVersion().isNewerThan(new VersionNumber("2.467"))
+                ? wc.goTo("user/alice/account/")
+                : wc.goTo("user/alice/configure");
```

Comment:
Same rational for simplification as the earlier comment.  When jenkins.version is greater than 2.467, the code won't ever run on an older Jenkins baseline.
```suggestion
        HtmlPage pg = wc.goTo("user/alice/account/");
```

Reply:
@MarkEWaite done